package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "nodejs_program",
+  "version": "1.0.0",
+  "description": "",
+  "module": "commonjs",
+  "main": "dist/app.js",
+  "scripts": {
+    "start": "node index.ts",
+    "dev": "nodemon --exec babel-node index.ts",
+    "build": "babel ./index.ts --out-dir dist --extensions \".js,.ts\" --copy-files --no-copy-ignored",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DmitrySkyrat/NodeJS_program.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/DmitrySkyrat/NodeJS_program/issues"
+  },
+  "homepage": "https://github.com/DmitrySkyrat/NodeJS_program#readme",
+  "dependencies": {
+    "@types/joi": "^17.2.3",
+    "csvtojson": "^2.0.10",
```

Comment:
Is it used? I thought we should do the next tasks from scratch without first homework.

Reply:
Thanks, I've forgotten to delete csvtojson